import-automation/import-progress-dashboard-frontend/client/src/serviceWorker.js

```diff
@@ -0,0 +1,141 @@
+// This optional code is used to register a service worker.
```

Comment:
It might simplify things to leave this out for now.

Reply:
removed